03/app.js

```diff
@@ -1,14 +1,34 @@
-console.log('DOM');
+console.log("DOM");
 
 const buttonSettings = {
-    attr: {
-        className: 'btn',
-        title: 'super button'
-    },
-    css: {
-        border: '1px solid #336699',
-        padding: '5px 20px',
-        color: '#444'
-    },
-    text: 'Click me!',
-}
\ No newline at end of file
+  attr: {
+    className: "btn",
+    title: "super button",
+  },
+  css: {
+    border: "1px solid #336699",
+    padding: "5px 20px",
+    color: "#444",
+  },
+  text: "Click me!",
+};
+
+const newButton = document.createElement("button");
+const parentForButton = document.querySelector(".parent-for-button");
+console.log(parentForButton);
+
+if (newButton) {
+  for (const set in buttonSettings) {
+    newButton.innerHTML = buttonSettings[set];
```

Comment:
Należałoby sprawdzić czy to co jest w `set` to np. `string` i wtedy wykonać odpowiednie działanie.

Reply:
Poprawilam, ale nie jestem pewna czy to o to chodzi?